nsv13/code/modules/events/overmap/lone_hunter.dm

```diff
@@ -0,0 +1,29 @@
+/datum/round_event_control/lone_hunter

```

Comment:
Please add a brief description of each event to the file.
EG:
//An event which will spawn a lone ship to hunt the crew (presumably, this is what this looks like but I cannot tell)

Reply:
Incorporated changes.